src/main/java/clipper2/engine/PolyPathNode.java

```diff
@@ -1,26 +1,28 @@
 package clipper2.engine;
 
+import clipper2.Nullable;
+import clipper2.core.Path64;
+
 import java.util.ArrayList;
 import java.util.List;
 
-import clipper2.core.Path64;
-
-public abstract class PolyPathBase implements Iterable<PolyPathBase> {
+public abstract class PolyPathNode implements Iterable<PolyPathNode> {
```

Comment:
Was this renamed like this in the parent project?

Reply:
Yes, the class was renamed.